src/dbwrapper.h

```diff
@@ -19,6 +19,17 @@
 static const size_t DBWRAPPER_PREALLOC_KEY_SIZE = 64;
 static const size_t DBWRAPPER_PREALLOC_VALUE_SIZE = 1024;
 
+struct LevelDbOptions : public leveldb::Options {
+    LevelDbOptions();
+
+    ~LevelDbOptions();
+
+    /** Sets block_cache and write_buffer_size since nCacheSize isn't known when
+     * LevelDbOptions ctor is called.
```

Comment:
I don't think this is strictly true? Or am I missing something? `nCacheSize` is passed into `CDBWrapper::CDBWrapper`, couldn't you pass it in like
```
CDBWrapper::CDBWrapper(const fs::path& path, size_t nCacheSize, bool fMemory, bool fWipe, bool obfuscate)
    : m_name{fs::PathToString(path.stem())},
      options(nCacheSize)
⋮

struct LevelDbOptions : public leveldb::Options {
    LevelDbOptions(size_t nCacheSize);
```

Reply:
Thanks!